tools/c7n_azure/c7n_azure/resources/sqldatabase.py

```diff
@@ -148,6 +148,88 @@ def _process_resource_set(self, resources, event=None):
         return result
 
 
+@SqlDatabase.filter_registry.register('data-masking-policy')
+class DataMaskingPolicyFilter(Filter):
+    """
+    Filter by the current data masking policy
```

Comment:
nit: just realized that all new SQL filters are pretty expensive to run.. I would add some message to recommend using them as a final filter (after tags etc if exists)

Reply:
Yeah and the server ones aren't as bad as these database ones since on top of everything else these are a child resource so we do servers -> databases -> data masking